playbook/app/pb_kits/playbook/pb_avatar/_avatar.scss

```diff
@@ -78,36 +78,6 @@ $avatar-sizes: (
           display: block;
         }
       }
-      [class^=pb_online_status] {
-        position: absolute;
-        right: 0;
-        top: 0;
-        bottom: auto;
-        left: auto;
-
-        &.size_xxs {
-          right: -5px;
-        }
-
-        &.size_xs {
-          right: -4px;
-        }
-
-        &.size_md {
-          top: auto;
-          bottom: 6px;
-        }
```

Comment:
I positioned this dot on the bottom right in a different way than we used to do it. 

Instead of 
```css
top: auto;
bottom: 6px;
left: auto;
right: 0;
```
I made it so it's:
```css
bottom: 0;
right: 4px;
```